src/dex-ui/pages/CreateProposal/CreateProposal.tsx

```diff
@@ -64,7 +73,8 @@ export const CreateProposal = (props: any) => {
           <Box width="600px">
             <Text textStyle="h3">Add New Token</Text>
             <Spacer padding="1rem" />
-            <AddNewToken title={title} handleTitleChange={handleTitleChange} />
+            {/* <AddNewToken title={title} handleTitleChange={handleTitleChange} /> */}
+            <AddNewText value={value} handleTitleChange={handleValueChange} />
```

Comment:
This change is only to test the Editor on create proposal

Reply:
We should dynamically change which form component renders (AddNewToken or AddNewText) based on the navigate location props that are provided from react-router. The docs for react-router can we viewed here: https://reactrouter.com/en/main.